doc/developing.rst

```diff
@@ -126,7 +122,114 @@ Building the HTML documentation
 
 To build the HTML documentation (output in ``doc/build/html``), run::
 
-  $ bin/sphinxbuilder
+  $ sphinx-build doc doc/build/html
+
+Developing Reg, Dectate or Importscan
+-------------------------------------
+
+If you need to adjust the sources of reg, dectate or importscan and test them
+together with morepath, they're available in the src directory.
+There you can edit them and test the changes with morepath, which uses them as
+dependencies when you install Morepath for developement as described above.
+
+Installing tox locally with with different Python versions
+----------------------------------------------------------
+
+If you want to check if Morepath works works with the supported Python versions
+and check if it pathes all Travis Ci tests, you can install tox locally.
+
+Tox is also used by Travis Ci on GitHub.
+
+First you should install all Python versions, which you want to test. The
+versions which are not installed will be skipped. At least you should install
+Python 3.5, which is required by flake8, coverage and doctests, and Python 2.7
+for testing Morepath with Python 2.
+
+You can install the Python versions, which are missing in your system with
```

Comment:
I think the section on pyenv should be removed, as it goes into too much detail on something that's not really essential (I have Pythons installed another way). Instead, say something like "One tool you can use to install multiple versions of Python is pyenv" with a link to its documentation.